debezium-connector-jdbc/src/test/java/io/debezium/connector/jdbc/CollectionNamingStrategyTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.Map;
+
+import org.apache.kafka.connect.errors.DataException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.debezium.connector.jdbc.util.DebeziumSinkRecordFactory;
+import io.debezium.connector.jdbc.util.SinkRecordFactory;
+import io.debezium.doc.FixFor;
+import io.debezium.sink.naming.CollectionNamingStrategy;
+import io.debezium.sink.naming.DefaultCollectionNamingStrategy;
+
+/**
+ * Tests for the {@link CollectionNamingStrategy} interface and implementations.
+ *
+ * @author Chris Cranford
+ */
+@Tag("UnitTests")
+public class CollectionNamingStrategyTest {
+
+    @Test
+    public void testDefaultTableNamingStrategy() {
+        final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(Map.of());
+        final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
+        final DefaultCollectionNamingStrategy strategy = new DefaultCollectionNamingStrategy();
+        assertThat(strategy.resolveCollectionName(factory.createRecord("database.schema.table"), config.getCollectionNameFormat()))
+                .isEqualTo("database_schema_table");
+    }
+
+    @Test
+    public void testCollectionNamingStrategyWithTableNameFormat() {
+        final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(Map.of(JdbcSinkConnectorConfig.COLLECTION_NAME_FORMAT, "kafka_${topic}"));
+        final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
+        final DefaultCollectionNamingStrategy strategy = new DefaultCollectionNamingStrategy();
+        assertThat(strategy.resolveCollectionName(factory.createRecord("database.schema.table"), config.getCollectionNameFormat()))
+                .isEqualTo("kafka_database_schema_table");
+    }
+
+    @Test
+    public void testDeprecatedTableNamingStrategyWithTableNameFormat() {
```

Comment:
@mfvitale You find tests for the deprecated option names here. Is that what you were looking for?

Reply:
I am refereeing more to the deprecation functionality it self. Checking the working message for example.